_posts/2015-03-04-aarsberetning.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: Årsberetning 2014/2015
+author: ekmartin
+
+---
+
+2015 har vært et interessant år for Webkom. Vi har hatt våre oppturerer og nedturer, selv om det definitivt har vært flest av førstnevnte. Nye prosjekter har blitt startet og gamle har blitt vedlikeholdt. Nettopp det å starte nye prosjekter er nok også noe av det vi har blitt enda flinkere på i løpet av året som har gått. Nesten all ny kode som skrives i Webkom blir åpen kildekode før eller senere, gjerne med en tilhørende solid testdekning. Dette vil vi fortsette med i løpet av 2015, og da mens vi skriver ny Abakus.no. Her har du en kort oppsummering av noe av det vi har drevet med siden forrige generalforsamling.
+
+## Blogg
+Bloggen du leser nå prøver vi å holde oppdatert med info om nye prosjekter og erfaringer vi opparbeider oss i løpet av vårt komitéarbeid. Vi vil på den måten prøve å oppnå at flere skal få nytte av det vi har lært. Et av målene våre er at alle i Webkom skal skrive minst én post i løpet av et skoleår.
+
+## census [<span class="fa fa-github"></span>](https://github.com/webkom/census)
+census er et nytt prosjekt som gir oss en oversikt over serverne våre. Det lever på
+[census.abakus.no](http://census.abakus.no). Man kan lese mer om census og hvordan
+det fungerer i [bloggposten](http://webkom.abakus.no/census/) vi skrev da det ble
+lansert. Det er verdt å merke seg at alle abakus-medlemmer kan bruke census til å
+holde oversikt over sine maskiner.
+
+## chewie [<span class="fa fa-github"></span>](https://github.com/webkom/chewie)
+chewie er det vi bruker til å produksjonssette prosjektene våre, og passer på at de alltid kjører en så ny versjon av koden som mulig. I tillegg til å gjøre dette automatisk har også chewie støtte for manuell produksjonssetting gjennom et webgrensesnitt. chewie kan du også lese mer om på den relevante bloggposten, [her](http://webkom.abakus.no/chewie/).
```

Comment:
chewie er det vi bruker til å produksjonssette prosjektene våre, og passer på at de alltid kjører siste versjon av koden. I tillegg til å gjøre dette automatisk har også chewie støtte for manuell produksjonssetting gjennom et web-grensesnitt. chewie kan du også lese mer om på [den relevante bloggposten](http://webkom.abakus.no/chewie/).